tree/develop/gogs_tools/testhooks

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/sh
```

Comment:
`#! /usr/bin/env sh` is more portable (see [here](https://en.wikipedia.org/wiki/Shebang_(Unix)#Portability))